tests/featureWriters/kernFeatureWriter2_test.py

```diff
@@ -694,6 +694,7 @@ def test_kern_LTR_and_RTL(self, FontClass):
 
             feature kern {
                 lookup kern_dflt;
+                lookup kern_ltr;
```

Comment:
If `kern_ltr` includes kerning between common characters (e.g. punctuation or digits) it would be wrong to register it for `arab`, otherwise it should be redundant and tiny bit slow but otherwise harmless.